### PR TITLE
Added Contact Form

### PR DESCRIPTION
**Pull request by ArkanKovanAbdullah**
## Added contact form: 

- added contact form (responsive)
- Made code a bit cleaner
- Took steps toward fully responsive

##

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -27,7 +27,6 @@ header {
 
 #section-one-one {
```
Comment (Alfredbis29) [inline styles.css:28]: - [ ] Very good design so far, you did a great job here but we can improve something to make it attractive to recruiters.
Here you can see how the width of the page is fixed but it should have width:100% to re-flow according to the screen width. :wink: 
Your Design :point_down: 
in


Figma Design :point_down: 
contactform

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @ArkanKovanAbdullah,
 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- Good readMe :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

 ## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -13,19 +13,24 @@
         <header>
             <nav class="be-flex">
                 <div class="navitems Logo" id="my-name"><a href="index.html">IDAE</a></div>
```
[the remark this answers is not in the document]
Comment (ArkanKovanAbdullah) [reply index.html:15]: Thanks a lot just fixed it.

File: styles.css (hunk 1 above)
Comment (ArkanKovanAbdullah) [reply styles.css:28]: Hi @Alfredbis29 
Many thanks. To be honest i thought this task was only for the contact form. But I appreciate your comment regarding this one. Please recheck, I guess you fall below the supposed width. which was 327. 
I will wait for your reply before submitting again. thanks...

Comment (ArkanKovanAbdullah) [reply styles.css:28]: You will face this issue on any other website if you fall below that width. 
for example check this image, Which is a image of the current page of github. 
29 09 2022_23 51 42_REC

**Review by levy002 — APPROVED** [review]
### Hello @ArkanKovanAbdullah 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @ArkanKovanAbdullah  👋, 

_This is an additional review as a result of the second opinion request._

 - I checked your project design and it looks nice and responsive in mobile version 👍, and I'm happy your project has been approved  :)

 Keep rocking 🚀!

**Pull request merged**